tests/yam/validate/validate_registered_addons.pm

```diff
@@ -0,0 +1,27 @@
+# SUSE's openQA tests
+#
+# Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+
+# Summary: Validate installed patterns against the list of additional patterns
+# selected to be installed during installation.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base 'consoletest';
+use strict;
+use warnings;
+use utils 'zypper_call';
+use testapi;
+
+sub run {
+    select_console 'root-console';
+
+    my @addon_list = split(/,/, get_var('ADDONS'));
+
+    # Confirm addons registered
+    zypper_call("search -t product");
+    foreach (@addon_list) { zypper_call("search -i -t product $_"); }
```

Comment:
this code checks via zypper the products (I guess it is kind of a verification for registration) but I would add below the check with suseconnect too.

Reply:
Added.